code/modules/mob/living/carbon/brain/brain.dm

```diff
@@ -2,16 +2,26 @@
 
 /mob/living/carbon/brain
 	languages = HUMAN
-	var/obj/item/container = null
+	var/obj/item/device/mmi/container = null

```

Comment:
container is never something other than a mmi, so might as well make its type a MMI so we don't have to do type checks.